antenna.c

```diff
@@ -561,7 +562,7 @@ get_route_area_forward_fromseg(NET net, ROUTE rt, SEG nseg, int layer,
 	    /* antenna area forward and reverse from that segment.	*/
 
 	    rt2 = rt->end.route;
-	    if (rt2 == NULL) return;  /* This should not happen */
+            assert( rt2 != NULL ) ;
```

Comment:
Nice catch. The compiler is right: this would return void for a function that expected to return a float.

Reply:
Okay, now I just need to figure out how to handle the pull request from the primary repository, since github is a mirror site for me. . .  any helpful hints?